packages/shared/src/relay/useGetPreviewImageUrlsWithPolling.ts

```diff
@@ -0,0 +1,52 @@
+import { useEffect } from 'react';
```

Comment:
the hook that wraps above method. comes with built-in polling and error reporting